---
title: ''
csl: mbio.csl #Get themes at https://github.com/citation-style-language/styles
fontsize: 11pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
  html_document:
    df_print: paged
geometry: margin=1.0in
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE, tidy=TRUE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)
library(here)
library(readxl)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=2, nsmall=2, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)

metadata <- read_excel(here("data/process/metadata.xlsx"), col_types = c("text", "numeric", "text", "text", "numeric", "text", "numeric", "text", "text", "text", "text", "numeric", "numeric")) #specify column types

```



\vspace{35mm}

# An osmotic laxative renders mice susceptible to prolonged *Clostridioides difficile* colonization and hinders clearance

\vspace{35mm}


Sarah Tomkovich^1^, Ana Taylor^1^, Jacob King^1^, Joanna Colovas^1^, Lucas Bishop^1^, Kathryn McBride^1^, Sonya Royzenblat^1^, Nicholas A. Lesniak^1^, Ingrid L. Bergin^2^, Patrick D. Schloss^1$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

1\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI, USA

2\. The Unit for Laboratory Animal Medicine, University of Michigan, Ann Arbor, MI, USA


\newpage
\linenumbers


## Abstract
(Modify depending on target journal, currently abstract submitted to World Microbe Forum) Antibiotics are a major risk factor for *Clostridioides difficile* infections (CDIs) because of their impact on the intestinal microbiome. However, non-antibiotic medications such as the ubiquitous osmotic laxative polyethylene glycol (PEG) 3350, also alter the microbiota, but whether PEG impacts CDI susceptibility and clearance is unclear. To examine how PEG impacts susceptibility, we treated C57Bl/6 mice with 5-day and 1-day doses of 15% PEG in the drinking water and then challenged the mice with *C. difficile* 630 spores. We used clindamycin-treated mice as a control because they consistently clear *C. difficile* within 10 days post-challenge (dpc). To examine how PEG treatment impacts clearance, we administered PEG for 1 day to clindamycin-treated, *C. difficile*-challenged mice either immediately following challenge or 3 dpc. We collected longitudinal stool samples to examine *C. difficile* levels in the stool via anaerobic culture and profiled the microbiota by 16S rRNA sequencing. PEG treatment alone was sufficient to render mice susceptible to CDI and 5-day PEG-treated mice remain colonized for up to 30 dpc. Additionally, 5-day PEG treated mice remained susceptible to CDI 10-days post treatment. In contrast, 1-day PEG treated mice were transiently colonized, clearing *C. difficile* within 7 dpc. Although 5-day PEG-treated mice exhibited prolonged *C. difficile* colonization, we saw no difference in histological inflammation between PEG- and clindamycin-treated mice. Additionally, administering PEG to mice after *C. difficile* challenge prolonged colonization up to 30 dpc in mice that received PEG immediately after challenge and 15 dpc in mice that received PEG 3 dpc. When we examined microbiota composition across our different treatment groups, we found decreased richness in the PEG-treated mice that exhibited prolonged *C. difficile* colonization. Importantly, there were increased Bacteroides and Enterobacteriaceae and decreased Lachnospiraceae and Oscillibacter in most of the PEG-treated mice with prolonged *C. difficile* colonization. Our findings suggest the osmotic laxative PEG 3350 alters the mouse microbiota and disrupts colonization resistance to *C. difficile*, as well as clearance in mice with a CDI. Considering that most hospitals recommend not performing *C. difficile* testing on patients taking laxatives and laxatives are used when administering fecal microbiota transplants via colonoscopy to patients with recurrent CDIs, further studies are needed to evaluate if laxatives impact human microbiota colonization resistance.

\newpage

## Introduction

Antibiotics are a major risk factor for *Clostridioides difficile* infections (CDIs) because they disrupt microbiota colonization resistance [@Britton2014]. However, antibiotics are not the only types of medications that disrupt the microbiota [@Maier2018; @LeBastard2017; @VichVila2020]. Although, other medications (proton pump inhibitors, osmotic laxatives, antimotility agents, and opioids) have been implicated as risk or protective factors for CDIs through epidemiological studies, whether the association is due to their impact on the microbiome is still unclear [@Oh2018; @Mora2012; @Nehra2018; @Krishna2013; @Tomkovich2019].

Many of the non-antibiotic medications associated with CDIs are known to modulate gastrointestinal motility leading to either increased or decreased colonic transit time, which in turn also strongly impacts microbiota composition and function [@Vandeputte2015; @VujkovicCvijin2020]. Stool consistency often serves as an approximation of intestinal motility [@Vandeputte2015]. Our group has shown that when *C. difficile* negative controls are separated into two groups based on stool consistency, there are shared microbiota features such as lower alpha diversity in samples from CDI patients and control patients with diarrhea compared to control samples that were *C. difficile* negative with non-diarrheal consistency [@Schubert2014]. These results led to a hypothesis that bacterial communities from patients experiencing diarrhea are susceptible to developing CDIs.

Osmotic laxatives can lead to diarrhea depending on the administered dose and temporarily disrupt the human intestinal microbiota [@Nagata2019]. The ubiquitous osmotic laxative, polyethylene glycol (PEG) 3350 is found in Miralax, Nulytely, and Golytely and is also commonly used as bowel preparation for colonoscopies. Interestingly, previous studies have shown that treating mice with PEG alone altered microbiota composition, reduced acetate and butyrate production, altered the mucus barrier, and rendered the mice susceptible to *C. difficile* infection [@Kashyap2013; @Ferreyra2014; @Tropini2018; @VanInsberghe2020]. The mucus barrier is thought to mediate protection from *C. difficile* infections by protecting intestinal epithelial cells from the toxins produced by *C. difficile* [@Olson2013; @Diebel2014]. Whether laxative administration results in more severe CDIs in mice and how long mice remain colonized with *C. difficile* after challenge is unclear.

Beyond susceptibility, PEG is also relevant in the context of treating recurrent CDIs via fecal microbiota transplant (FMT) where a healthy microbiota is administered to the patient to restore colonization resistance. For FMTs that are delivered via colonoscopy, patients typically undergo bowel preparation by taking an osmotic laxative prior to the procedure. Many of the FMT studies to date rationalize the use of laxatives [@vanNood2013; @Razik2017; @Postigo2012] based on a 1996 case study with 2 pediatric patients where the authors suggested in the discussion that the laxative may help flush *C. difficile* spores and toxins from the intestine [@Liacouras1996].

Our group has used C57BL/6 mice to characterize how antibiotics disrupt the microbiota and influence *C. difficile* susceptibility and clearance [@Schubert2015; @Jenior2017; @Tomkovich2020]. Although, two groups have now shown PEG treatment alone renders mice susceptible to *C. difficile* [@Ferreyra2014; @VanInsberghe2020], these studies have raised additional questions regarding the dynamics and severity of infection as well as the role of laxative treatment in *C. difficile* clearance. Addressing these questions will better inform how we think about laxatives in the context of CDIs. Here, we characterized how long PEG-treated mice remain susceptible, whether PEG treatment results in more sustained *C. difficile* colonization and severe CDI and  than mice treated with clindamycin, and if PEG treatment after colonization can promote *C. difficile* clearance.

## Results

**5-day laxative treatment led to prolonged *C. difficile* colonization in mice.** Building off of previous work that showed treating mice with the osmotic laxative, PEG 3350, rendered mice susceptible to *C. difficile* colonization [@Ferreyra2014; @VanInsberghe2020], we decided to test how long *C. difficile* colonization is sustained and how long PEG-treated mice remain susceptible to *C. difficile*.  We compared three groups of mice treated with PEG 3350 to one group of mice treated with our standard 10 mg/kg clindamycin treatment, which temporarily renders the mice susceptible to *C. difficile* colonization, with mice typically clearing *C. difficile* within 10 days post-challenge [@Tomkovich2019; @Tomkovich2020]. All three groups of PEG-treated mice were administered a 15% PEG solution in the drinking water for 5-days, 1. received no additional treatment, 2. was also treated with clindamycin, and 3. was allowed to recover for 10 days prior to challenge (Fig. 1A). PEG treatment resulted in weight loss in all 3 groups of PEG-treated mice relative to their baseline weights, with the greatest change in weight observed on the fifth day of PEG treatment. The mice recovered most of the lost weight by five days after treatment (Fig. 1B). After either the PEG, clindamycin, or PEG and clindamycin treatment all mice were challenged with 10^5^ *C. difficile* 630 spores (Fig. 1A). All treatments rendered mice susceptible to *C. difficile* colonization. However, PEG-treated mice remained colonized with *C. difficile* at a high level through thirty days post-challenge (Fig. 1C). In contrast, the clindamycin-treated mice cleared *C. difficile* within ten days post-challenge (Fig. 1C). Therefore, PEG treatment led to sustained colonization in contrast to clindamycin mice that naturally cleared *C. difficile* within ten days post-challenge. 

Notably, we also found PEG-treated mice were still susceptible to *C. difficile* colonization after a 10-day recovery period, although *C. difficile* was not detectable in most of the group in the initial five days post-challenge (Fig. 1C, S1A). One mouse was found dead on the 6th day post-challenge, presumable due to *C. difficile* as the bacteria became detecable in that mouse on the 4th day post-challenge (Fig. S1A, mouse 10). From 8 days post-challenge onward, the density of *C. difficile* stabilized in the 10-day recovery group and remained high through 20-30 days post-challenge (Fig. 1C). Thus, osmotic laxative treatment alone was sufficient to render mice susceptible to prolonged *C. difficile* colonization and PEG-treated mice remained susceptible through ten days post-treatment.

``` {r 5-day PEG fecal microbiota}
f_d_permanova <- read_tsv(here("data/process/5_days_PEG_permanova_stools.tsv"))
f_d_sign_contrib <- f_d_permanova %>% arrange(desc(R2)) %>%
  filter(!effects == "Total") %>% #Remove total since it is not one of the variables
  filter(`Pr(>F)` < 0.05) #Only select significant contributors

#List of variables that significantly explained variation in stool samples
f_d_top_effects <- f_d_sign_contrib %>%
  pull(effects)
#Sum of R2 values for variables that significantly explained variation in stool samples
f_d_top_R2_sum <- f_d_sign_contrib %>%
  select(R2) %>% sum
#P-value for variables that significantly explained variation in stool samples
f_d_top_p <- f_d_sign_contrib %>%
  select(`Pr(>F)`) %>% max # Both p values are the same, will be rounded in text

#Number of experiments represented in 5-day subset
#Use same code from five_day_PEG_subset() in code/utilities.R to determine number of samples we had C. difficile CFU counts for per day
five_day_mice <- metadata %>% filter(group == "C" & exp_num %in% c("M3","M4", "M5", "M8")| #Only use C mice from these experiments. Allocated groups to figures based on paper outline.
             group == "WM" & exp_num %in% c("M3","M4", "M5", "M8")|
             group == "WMC" & exp_num %in% c("M3","M4")|
             group == "WMR" & exp_num %in% c("M5","M6"))
#Total number of experiments in 5-day subset
five_day_total_exp <- five_day_mice %>%
  filter(!duplicated(exp_num)) %>% #Remove duplicate experiment numbers
  tally() #Total number of unique experiments

#PEG impacted genera
PEG_genera <- read_tsv(here("data/process/5_days_PEG_genus_PEG_paired.tsv")) %>%
  filter(p.value.adj < 0.05) %>% #Limit to significant genera
  tally()

#Genera that were different between treatment groups
f_d_group_genera <- read_tsv(here("data/process/5_days_PEG_genus_group_stools.tsv")) %>%
  filter(p.value.adj < 0.05)  #Select only significant p-values
f_d_group_total_genera <- f_d_group_genera %>%
  filter(!duplicated(genus))  %>% #Remove genera that appear multiple times
  tally()
#Genera that were different between treatment groups over multiple days
f_d_group_multi_day <- f_d_group_genera %>%
  filter(p.value.adj < 0.05) %>% #Select only significant p-values
  group_by(genus) %>%
  tally() %>%
  filter(n > 1) %>% #select genera that vary over multiple time points = n > 1
  arrange(desc(n)) %>%  #Rank by how many time each genera shows up
  nrow() #24 genera includes Peptostreptococcaceae and Unclassified

```
**5-day laxative treatment differentially disrupted the fecal microbiota compared to clindamycin treatment.** Since osmotic laxatives and clindamycin have previously been shown to disrupt the murine microbiota [@Kashyap2013; @Ferreyra2014; @Tropini2018; @VanInsberghe2020], we hypothesized the different *C. difficile* colonization dynamics between mice treated with the osmotic laxative or clindamycin were due to the two drugs having differential effects on the microbiota. We profiled the stool microbiota over time by sequencing the V4 region of the 16S rRNA gene to compare changes across treatment groups. We found time and treatment group explained half of the observed variation between fecal communities with most of the remaining variation explained by interactions between treatment group and other experimental variables including time, cage effects, and sequencing preparation plate (PERMANOVA combined R^2^ = `r f_d_top_R2_sum`, *P* `r format.pval(f_d_top_p, eps = 0.001)`, Fig. 2A, Data Set S1, Sheet X). Cage effects refer to the well-documented phenomenon that mice housed in the same cages have similar microbial communities due to coprophagy [@Nguyen2015]. We tried to minimize the impact of cage effects on our experiment by breaking up cagemates when assigning mice to treatment groups. Importantly, although we conducted a total of `r five_day_total_exp` separate experiments, the experiment number and its interaction with treatment group was not one of the variables that significantly explained the observed variation in fecal communities (Data Set S1, Sheet X). None of the treatment groups recovered to their baseline community structure either 10 or 30 days post-challenge suggesting other community features besides recovery to baseline were responsible for the prolonged *C. difficile* colonization in PEG-treated mice (Fig. 2B).

Since none of the communities completely recovered in the follow-up period after treatments, we next profiled community diversity and composition. We examined the alpha diversity dynamics by calculating the communities' Shannon diversity. Although both clindamycin and PEG treatments decreased diversity, Shannon diversity was lower in the groups of mice that received PEG treatment compared to those that received clindamycin through thirty days post-challenge (Fig. 2C). We next examined the bacterial genera that shifted after PEG treatment by comparing the baseline samples of mice treated with only PEG to samples from the same mice one day post-PEG-treatment. We found `r PEG_genera` genera that were altered by PEG treatment (Data Set S1, Sheet X). The majority of the bacterial relative abundances decreased after the PEG treatment, but the relative abundance among members of the *Enterobacteriaceae* and *Bacteroides* increased. The increase in *Bacteroides* relative abundance was unique to PEG treated mice, as the *Bacteroides* relative abundance actually decreased in clindamycin treated mice (Fig. 2D). Finally, we identified the genera whose relative abundance differed across treatment groups over multiple time points. Of the `r f_d_group_total_genera` genera that were different between treatment groups, `r f_d_group_multi_day` genera were different over multiple time points  (Fig. 2E, Data Set S1, Sheet X). Thus, PEG had a significant impact on the fecal microbiota that was maintained over time and was distinct from clindamycin treatment.

Interestingly, *C. difficile* was not immediately detectable in the stools of the PEG-treated mice that were allowed to recover for 10 days prior to  challenge. We decided to examine if there were genera that changed during the post-challenge period when the group median *C. difficile* shifted from undetectable at 1 day post-challenge to detectable in the stool samples with the density stabilizing around 8 days post-challenge (Fig. S1A). We found no bacteria were significantly different over the two time periods after multiple hypothesis correction (Data Set S1, Sheet X). However, there was also wide variation between individual mice regarding when *C. difficile* became detectable (Fig. S1A) as well as the relative abundances of bacterial genera in the communities (Fig. S1B). For example, two mice had a high relative abundance of *Enterobacteriaceae* throughout the post-challenge period and this corresponded to mouse 10, which died on the sixth day post-challenge and mouse 11, where *C. difficile* was present at a high density from the 4th day post-challenge onward (Fig. S1B). While we did not identify a clear signal to explain the delayed appearance of *C. difficile* in the 5-day PEG mice that were allowed to recover for 10 days prior to challenge, the delay is striking and could reflect changes in microbial activity or metabolites that were not examined in this study.

``` {r 5-day PEG tissue microbiota}
f_d_permanova_t <- read_tsv(here("data/process/5_days_PEG_permanova_tissues.tsv"))
f_d_sign_contrib_t <- f_d_permanova_t %>% arrange(desc(R2)) %>%
  filter(!effects == "Total") %>% #Remove total since it is not one of the variables
  filter(`Pr(>F)` < 0.05) #Only select significant contributors

#List of variables that significantly explained variation in tissue samples
f_d_top_effects_t <- f_d_sign_contrib_t %>%
  pull(effects)
#Sum of R2 values for variables that significantly explained variation in tissue samples
f_d_top_R2_sum_t <- f_d_sign_contrib_t %>%
  select(R2) %>% sum
#P-value for variables that significantly explained variation in tissue samples
f_d_top_p_t <- f_d_sign_contrib_t %>%
  select(`Pr(>F)`) %>% max # P value will be rounded in text

#Function to find which significant otus/genera/families are shared
intersect_all <- function(a,b,...){
  Reduce(intersect, list(a,b,...))
}
#Number of genera different in all 3 types of tissue samples
kw_genus_cecum <- read_tsv(here("data/process/5_days_PEG_genus_group_cecum.tsv")) %>% 
  filter(p.value.adj < 0.05) %>% pull(genus)
kw_genus_proximal_colon <- read_tsv(here("data/process/5_days_PEG_genus_group_proximal_colon.tsv")) %>%
  filter(p.value.adj < 0.05) %>% pull(genus)
kw_genus_distal_colon <-  read_tsv(here("data/process/5_days_PEG_genus_group_distal_colon.tsv")) %>% 
  filter(p.value.adj < 0.05) %>% pull(genus)
# of shared genera with significantly different relative abundances between groups
shared_tissue_genera <- length(intersect_all(kw_genus_cecum, kw_genus_proximal_colon, kw_genus_distal_colon))

tissue_genera_max_p <- read_tsv(here("data/process/5_days_PEG_genus_group_cecum.tsv")) %>% 
  add_row(read_tsv(here("data/process/5_days_PEG_genus_group_proximal_colon.tsv"))) %>% 
  add_row(read_tsv(here("data/process/5_days_PEG_genus_group_distal_colon.tsv"))) %>% 
  filter(p.value.adj < 0.05) %>% 
  #Limit to genus found in all 3 types of tissues
  filter(genus %in% intersect_all(kw_genus_cecum, kw_genus_proximal_colon, kw_genus_distal_colon)) %>%
  select(p.value.adj) %>% max()

```
**5-day laxative treatment did not promote more severe CDIs despite altering the mucosal microbiota.** Given the findings from a previous study that demonstrated PEG treatment disrupts the mucus layer and alters the immune response in mice [@Tropini2018], we decided to examine the impact of PEG treatment on the mucosal microbiota and CDI severity. To evaluate the mucosal microbiota, we sequenced snips of tissue collected from the cecum, proximal colon, and distal colon. Similar to what was observed with the stool samples, alpha diversity was lower in the PEG-treated mice compared to clindamycin treated mice (Fig. 3A). Alpha diversity continued to increase over time with the PEG-treated mice collected at 20 and 30 days post-challenge (Fig. 3A, Data Set S1, Sheet X). Group, time point, and their interactions with other variables (cage, experiment number, and sample type) explained the majority of the variation observed in mucosal communities (PERMANOVA combined R^2^ = `r f_d_top_R2_sum_t`, *P* `r format.pval(f_d_top_p_t, eps = 0.05)`, Fig. 3B, Data Set S1, Sheet X). We saw the greatest difference in the relative abundance of the mucosal microbiota between treatment groups (clindamycin, 5-day PEG, and 5-day PEG plus clindamycin) at 6 days post-challenge with `r shared_tissue_genera` genera that were significantly different (*P* `r format.pval(tissue_genera_max_p, eps = 0.05)) in all three of the tissue types we collected (cecum, proximal colon, and distal colon; Fig. S2A, Data Set S1, Sheet X). Interestingly, *Peptostreptococcaceae* (the family with a sequence that matches *C. difficile*) was one of the genera that had a significant difference in relative abundance between treatment groups at 6 days post-challenge. This population was primarily only present in the 5-day PEG treatment group of mice and decreased in the proximal and distal colon tissues over time (Fig. S2B). By 30 days post-challenge, only the relative abundances of *Bacteroides*, *Clostridiales*, *Firmicutes*, and *Ruminococcaceae* were different between treatment groups and only in the cecum tissues (Fig. 3C, Fig. 2E, Data Set S1, Sheet X). Thus, PEG treatment had a significant impact on the mucosal microbiota and we detected *C. difficile* sequences in the cecum, proximal colon, and distal colon tissue communities.

Next, we examined the severity of *C. difficile* challenge by evaluating cecum and colon H&E stained histopathology [@Reeves2011] and found there was no difference in cecum and colon scores between clindamycin and PEG-treated mice that were challenged with *C. difficile* at 4 days post-challenge (Fig. 3D), the time point typically examined in *C. difficile* 630 challenged mice [@Dieterle2020]. We also looked at 6 days post-challenge because that was when there was a large difference in *C. difficile* density between PEG- and clindamycin-treated mice (Fig. 1C). Although, there was a slight difference in the colon between PEG and clindamycin-treated mice, there was not a signifant difference in the cecum and the overall score was relatively low (1.5-2.5 out of 12, Fig. 3E). Therefore, although PEG treatment had a disruptive effect on the mucosal microbiota, the impact of *C. difficile* 630 challenge on the cecum and colon was similar between PEG and clindamycin treated mice.

***C. difficile* challenge did not have a synergistic disruptive effect on the microbiota of PEG-treated mice.** Because *C. difficile* itself can have an impact on the microbiota [@Jenior2018], we also sequenced the tissue and stools of mock-challenged clindamycin and 5-day PEG treated mice. Examining the stools of the mock-challenged mice revealed similar bacterial disruptions as the *C. difficile* challenged mice (Fig. S3A-C). Similarly, there was no difference between the tissues of mock and *C. difficile* challenged mice (Fig. S3D-F). Thus, most of the microbiota alterations we observed in the PEG-treated mice were a result of the laxative and not an interaction between the laxative and *C. difficile*.
``` {r 1-day PEG stool microbiota}

#Genera that were different between treatment days
#Genera that were changed by treatment
pt_d1 <- read_tsv(here("data/process/1_Day_PEG_genus_stats_BtoD1.tsv")) %>%
  filter(p.value.adj < 0.05)
#Genera that were still different at 7 days post-challenge  
pt_d7 <- read_tsv(here("data/process/1_Day_PEG_genus_stats_BtoD7.tsv")) %>%
  filter(p.value.adj < 0.05)
pt_d7_genera <- pt_d7 %>% pull(genus)

recovered_genera <- pt_d1 %>%
  filter(!genus %in% pt_d7_genera) %>% #Remove genera that were still different at day 7
  nrow() #Tally number of rows

```
**1-day laxative treatment resulted in transient *C. difficile* colonization and minor microbiota disruption.** Next, we examined how a shorter osmotic laxative perturbation would impact the microbiome and susceptibility to *C. difficile*. We administered either a 1-day PEG treatment, a 1-day PEG treatment with a 1-day recovery period, or clindamycin to mice before challenging them with *C. difficile* (Fig. 3A). In contrast to the 5-day PEG treated mice, the 1-day PEG treated mice were only transiently colonized and cleared *C. difficile* by 7 days post-challenge (Fig. 3B). The stool communities of PEG-treated mice were also only transiently disrupted, with Shannon diversity recovering by 7 days post-challenge (Fig. 3C-D). We found the relative abundances of `r recovered_genera` genera were impacted by treatment, but recovered close to baseline levels by 7 days post-challenge including *Enterobacteriaceae*, *Clostridiales*, *Porpyromonadaceae*, and *Ruminococcaceae* (Fig. 3E, Data Set S1, Sheet X). These findings suggest the duration of the PEG treatment was relevant, with shorter treatments resulting in a transient loss of *C. difficile* colonization resistance.

``` {r Post-CDI PEG stool microbiota}

#Genera that were different between treatment groups
post_group_genera <- read_tsv(here("data/process/post_CDI_PEG_genus_group_stools.tsv")) %>%
  filter(p.value.adj < 0.05)  #Select only significant p-values
post_group_total_genera <- post_group_genera %>%
  filter(!duplicated(genus))  %>% #Remove genera that appear multiple times
  tally() #26 total genera

#Genera that were different between treatment groups over multiple days
post_group_multi_day <- post_group_genera %>%
  filter(p.value.adj < 0.05) %>% #Select only significant p-values
  group_by(genus) %>%
  tally() %>%
  filter(n > 1) %>% #select genera that vary over multiple time points = n > 1
  arrange(desc(n)) %>%  #Rank by how many time each genera shows up
  nrow() #24 genera includes Peptostreptococcaceae and Unclassified

```
**Post-CDI laxative treatment disrupted clearance in clindamycin-treated mice regardless of whether an FMT was also administered.** Since a 1-day PEG treatment resulted in a more mild microbiota perturbation, we decided to use the 1-day treatment to examine the hypothesis that PEG helps to flush *C. difficile* spores from the intestine. To examine the impact of PEG treatment on *C. difficile* clearance, we treated 4 groups of mice with clindamycin and then challenged all mice with *C. difficile* before administering the following treatments: no additional treatment, 1-day PEG immediately after challenge, and 1-day PEG treatment 3 days after challenge followed by either administration of an FMT or PBS solution by oral gavage (Fig. 5A). Contrary to our hypothesis, all groups of mice that received PEG exhibited prolonged *C. difficile* colonization (Fig. 5B). Next we examined how post-CDI PEG treatment impacted microbiota diversity. Alpha diversity was lower in the PEG-treated mice with the exception of the PEG-treated mice that were administered an FMT (Fig 5C-D).

We were also interested in exploring whether PEG might help with engraftment in the context of FMTs. The FMT appeared to partially restore Shannon diversity but not richness (Fig. 5C-D). Similarly, we saw some overlap between the communities of mice that received FMT and the mice treated with only clindamycin after 5 days post-challenge (Fig. 6A). The increase in Shannon diversity suggests that the FMT did have an impact on the microbiota, despite seeing prolonged *C. difficile* colonization in the FMT treated mice. However, only the relative abundances of *Bacteroidales* and *Porphyromonadaceae* consistently differed between the mice received either an FMT or PBS gavage (Fig. 6B), suggesting the FMT only restored a couple of genera. Overall, we found the relative abundances of `r post_group_multi_day` genera were different between treatment groups over multiple timepoints. For example, the relative abundance of *Akkermansia* was increased and the relative abundances of *Ruminococcaceae*, *Clostridiales*, *Lachnospiraceae*, and *Oscillibacter* were decreased in mice that received PEG after *C. difficile* challenge relative to clindamycin treated mice (Fig. 6C). In sum, administering PEG actually prolonged *C. difficile* colonization, including in mice that received an FMT, which only restored 2 bacterial genera.

``` {r ml results}
#AUC results
auc_results <-  read_csv(here("results/performance_results.csv")) %>%
  mutate(taxa_level = "genus") %>%  
  group_by(method) %>%
  summarize(median = median(AUC)) %>%
  arrange(desc(median))
#Highest AUC value
max_auc <- auc_results %>% select(median) %>% max()

#To Do: Insert minimum Percent identity as variable name once blast results to Muribaculum have been combined into 1 file


```
**Five-day post-challenge community data can predict which mice that will have prolonged *C. difficile* colonization.** After identifying bacteria associated with the 5-day, 1-day and post-CDI 1-day PEG treatments, we decided to examine the bacteria that influenced prolonged *C. difficile* colonization. We trained 3 types of machine learning models (random forest, logistic regression, and support vector machine) with bacterial community data from 5 days post-challenge to predict whether the mice were still colonized with *C. difficile* 10 days post-challenge. We chose 5 days post-challenge because that was the earliest time point where we would see a treatment effect in the mice that were given 1-day PEG treatment three days after challenge. The random forest model had the highest performance (AUROC = `r max_auc`, Data Set S1, Sheet X), so we next performed permutation importance to examine the bacteria that were the top contributors to the random forest model predicting prolonged *C. difficile *colonization. We selected the top 10 bacteria contributing to our models performance (Fig. 7A) and examined their relative abundance at 5 days post-challenge, the time point used to predict *C. difficile* colonization status on day 10 (Fig. 7B). Next, we focused on the 5 genera that had a greater than 1 % relative abundance in either the cleared or colonized mice and examined how the bacteria changed over time. We found *Enterobacteriaceae* and *Bacteroides* tended to have a higher relative abundance, the relative abundance of *Akkermansia* was initially decreased and then increased, and *Porphyromonadaceae* and *Lachnospiraceae* had a lower relative abundance in the mice with prolonged colonization compared to the mice that cleared *C. difficile* (Fig. 7C). Together these results suggest a combination of low and high abundance bacterial genera influence the prolonged colonization observed in 5-day PEG and post-CDI PEG treated mice.

Previous work examining the impact of PEG on the murine microbiota found that PEG treatment resulted in the permanent loss of *Muribaculum intestinale* [@Tropini2018]., which is classified as *Porphyromonadaceae* by the Ribosomal Database Project (RDP) database [@Vornhagen2020]. *Porphyromonadaceae* was a top feature in the random forest model predicting prolonged *C. difficile* colonization. We identified 4 OTUs that had 92-96% identity to *Muribaculum intestinale* and examined their abundance in mice that either cleared or were still colonized with *C. difficile* at 10 days post-challenge. While all of the OTUs, were decreased by PEG and clindamycin treatment, there was some recovery in the mice that cleared (Fig. S4A). We also examined other *Porphyromonadaceae* and *Lachnospiraceae* OTUs since these were the 2 genera that were important to our classification model and contained multiple OTUs that were different at 5 days post-challenge between mice that either cleared or remained colonized with *C. difficile* by 10 days post-challenge (Data Set S1, sheet X). While individual *Porphyromonadaceae* and *Lachnospiraceae* OTUs tended to be more abundant in the mice that clear *C. difficile* relative to the mice that exhibit prolonged colonization (Fig. S4B-C), there is no single OTU that fits the pattern we observed at the genus level (Fig. 7C), suggesting multiple *Porphyromonadaceae* and *Lachnospiraceae* OTUs influenced *C. difficile* clearance. Overall, our results suggest that specific bacterial community differences explain the prolonged *C. difficile* colonization we observed in 5-day PEG and post-CDI 1-day PEG treated mice.

## Discussion

* Summary of major findings (Fig. 8A)

* Discussion of prolonged persistence. *C. difficile* sequences detected in tissue samples. Association with mucin-degrading bacteria suggested by recent papers.

* Discuss why we might not have observed more severe histology in PEG mice relative to clindamycin-treated mice
  + Antibiotics may also impact mucus layer
  + Strain of bacteria used

* Protective bacteria missing in PEG-treated mice

* Discuss what these findings might mean for human patients (Fig. 8B)
  + What's known regarding laxatives and susceptibility to CDIs
  + Clinical trial of PEG, results never posted [@Dieterle2018]
  + Relevance to human FMTs? Unclear what the best administration route is because there have been no studies designed to evaluate the best administration route for FMTs.


## Conclusions

## Acknowledgements
We thank members of the Schloss lab for feedback on planning the experiments and data presentation. We thank Andrew Henry for help with media preparation and bacterial culture and the Microbiology and Immunology department's postdoc association writing group members for their feedback on manuscript drafts. We also thank the Unit for Laboratory Animal Medicine at the University of Michigan for maintaining our mouse colony and providing the institutional support for our mouse experiments. Finally, we thank Kwi Kim, Austin Campbell, and Kimberly Vendrov for their help in maintaining the Schloss lab's anaerobic chamber. This work was supported by the National Institutes of Health (U01AI124255). ST was supported by the Michigan Institute for Clincial and Health Research Postdoctoral Translation Scholars Program (UL1TR002240 from the National Center for Advancing Translational Sciences).

## Materials and Methods
* Histopathology [@Theriot2011]

\newpage
```{r sample_size for figure legends}

#Use same code from five_day_PEG_subset() in code/utilities.R to determine number of samples we had C. difficile CFU counts for per day
five_day_mice <- metadata %>% filter(group == "C" & exp_num %in% c("M3","M4", "M5", "M8")| #Only use C mice from these experiments. Allocated groups to figures based on paper outline.
             group == "WM" & exp_num %in% c("M3","M4", "M5", "M8")|
             group == "WMC" & exp_num %in% c("M3","M4")|
             group == "WMR" & exp_num %in% c("M5","M6"))
five_d_cfu <- five_day_mice %>%
  filter(sample_type == "stool") %>% #Select only stool samples
  filter(day %in% c(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30)) %>%  #select time points on plot
  filter(!is.na(avg_cfu)) %>% #Remove rows with NA values
  group_by(day) %>%
  count()
#Figure out min & max number of samples we had cfu counts for
five_d_cfu_n_min <- min(five_d_cfu$n)
five_d_cfu_n_max <- max(five_d_cfu$n)

#Figure out which experiments eneded at 4, 6, or 30 days post-challenge
five_day_mice_samples <- five_day_mice %>% 
  group_by(exp_num, day, sample_type) %>% 
  tally()
#Figure out number of stool samples from 4,6, 10, 20 and 30 days post-challenge
five_day_mice_samples <- five_day_mice %>% 
  filter(sample_type == "stool" & day %in% c(4, 6, 10, 20, 30)) %>% 
  group_by(day) %>% 
  tally()

```

\includegraphics{figure_1.pdf}

**Figure 1. 5-day PEG treatment prolongs susceptibility and mice become persistently colonized with *C. difficile*.** A. Setup of the experimental time line for experiments with 5-day PEG treated mice consisting of 4 treatment groups. 1. Clindamycin was administered at 10 mg/kg by intraperitoneal injection. 2. 15% PEG 3350 was administered in the drinking water for five days. 3. 5-day PEG plus clindamycin treatment. 4. 5-day PEG plus 10-day recovery treatment. All treatment groups were then challenged with 10^5^ *C. difficile* 630 spores. A subset of mice were euthanized on either 4 or 6 days post-challenge and tissues were collected for histopathology analysis, the remaining mice were followed through 20 or 30 days post-challenge. B. Weight change from baseline weight in groups after treatment with PEG and/or clindamycin, followed by *C. difficile* challenge. C. *C. difficile* CFU/gram stool measured over time (N = `r five_d_cfu_n_min`-`r five_d_cfu_n_max` mice per time point) via serial dilutions. The black line represents the limit of detection for the first serial dilution. CFU quantification data was not available for each mouse due to stool sampling difficulties (particularly the day the mice came off of the PEG treatment) or early deaths. For B-C, lines represent the median for each treatment group and circles represent samples from individual mice. Asterisks indicate time points where the weight change or CFU/g was significantly different between groups by the Kruskal-Wallis test with Benjamini-Hochberg correction for testing multiple time points. The data presented are from a total of `r five_day_total_exp` separate experiments.
\newpage

\includegraphics{figure_2.pdf}
**Figure 2. 5-day PEG treatment disrupts the stool microbiota for a longer amount of time compared to clindamycin-treated mice.** A. Principal Coordinate analysis (PCoA) of Bray-Curtis distances from stool samples collected throughout the experiment. For A and C, each circle represents a sample from an individual mouse and the transparency of the symbol corresponds to the day post-challenge. B. Bray-Curtis distances of stool samples collected on either day 10 or 30 post-challenge relative to the baseline sample collected for each mouse (before any drug treatments were administered). C. Shannon diversity in stool communities over time. The line indicates the median value for each treatment group. The colors of the symbols and lines represent the four treatment groups. D. 14 of the `r f_d_group_total_genera` genera affected by PEG treatment  (Data Set S1, sheet X). The symbols represent the median relative abundance for a treatment group at either baseline (open circle) or 1-day post treatment (closed circle). Data from the 5-day PEG and 5-day PEG plus 10-day recovery groups were analyzed by paired Wilcoxan signed-rank test with Benjamini-Hochberg correction for testing all identified genera. The clindamycin and 5-day PEG plus clindamycin treatment groups are shown for comparison. E. 6 of the `r f_d_group_multi_day` genera that were significantly different between the four treatment groups over multiple time points. Differences between treatment groups were identified by Kruskal-Wallis test with Benjamini-Hochberg correction for testing all identified genera. The gray vertical line (D) and horizontal vertical lines (E) indicate the limit of detection.
\newpage

\includegraphics{figure_3.pdf}
**Figure 3. 5-day PEG treatment does not result in more severe CDIs, although mucosal microbiota is altered.** A. Shannon diversity in cecum communities over time. The line indicates the median value for each treatment group. The colors of the symbols and lines represent the four treatment groups. A similar pattern was observed with the proximal and distal colon communities (Data Set S1, sheet X-X). B. PCoA of Bray-Curtis distances from mucosal samples collected throughout the experiment. Circles, triangles, and squares indicate cecum, proximal colon, and distal colon communities, respectively. For A-B, transparency of the symbol corresponds to the day post-challenge that the sample was collected. C. The median relative abundance of the 4 genera that were significantly different between the cecum communities of different treatment groups on day 6 and day 30  (Data Set S1, sheet X). The gray vertical line indicate the limit of detection. D-E. The histopathology summary scores from cecum and colon H&E stained slides. The summary score is the total score based on evaluation of edema, cellular infiltration, and inflammation. Each category is given a score ranging from 0-4, thus the maximum possible summary score is 12. The tissue for histology was collected at either 4 (D) or 6 (E) days post-challenge with the exception that one set of 5-day PEG treated mock-challenged mice were collected on day 0 post-challenge (first set of open circles in D). Histology data were analyzed with the Kruskal-Wallis test followed by pairwise Wilcoxon comparisons with Benjamini-Hochberg correction.
\newpage

``` {r figure_4_permanova}
one_day_permnova_results <- read_tsv(here("data/process/1_day_PEG_permanova.tsv"))
one_day_top_2_contrib <- one_day_permnova_results %>% arrange(desc(R2)) %>%
  top_n(3, R2)
one_day_top_contrib_name = (one_day_top_2_contrib[2,1])
one_day_sec_contrib_name = (one_day_top_2_contrib[3,1])
top_R2 = one_day_top_2_contrib[2,6]
sec_R2 = one_day_top_2_contrib[3,6]


#Use same code from one_day_PEG_subset() in code/utilities.R to determine number of samples per day
one_day_samples_per_day <- metadata %>%
  filter(group == "C" & exp_num %in% c("M6")| #Only use C mice from this experiments. Allocated groups to figures based on paper outline.
             group == "1RM1" & exp_num %in% c("M6R")| #Had to differentiate experiment 6 from 6R in the metadata to create unique_mouse_id that wouldn't overlap for the M1 & 1RM1 mice that are both labeled with mouse_ids that are #s1-6
             group == "M1" & exp_num %in% c("M6")) %>%
  filter(day %in% c(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10)) %>%  #select time points on plot
  filter(!is.na(avg_cfu)) %>% #Remove rows with NA values
  group_by(day) %>%
  count()
#To Do: this code is not specific for CFU values or the time points shown on panel B
one_day_sample_min <- min(one_day_samples_per_day$n)
one_day_sample_max <- max(one_day_samples_per_day$n)


```
\includegraphics{figure_4.pdf}
**Figure 4. 1-day PEG treatment renders mice susceptible to transient *C. difficile* colonization.** A. Setup of the experimental time line for the 1-day PEG treated  mice consisting of 3 treatment groups. 1. Clindamycin was administered at 10 mg/kg by intraperitoneal injection. 2. 15% PEG 3350 was administered in the drinking water for 1 day. 3. 1-day PEG plus 1-day recovery. The three treatment groups were then challenged with 10^5^ *C. difficile* 630 spores. B. *C. difficile* CFU/gram stool measured over time (N = `r one_day_sample_min`-`r one_day_sample_max` mice per time point) via several dilutions. The black dotted line represents the limit of detection for the first serial dilution. Asterisks indicate time points where the CFU/gram was significantly different between treatment groups by Kruskall-Wallis test with Benjamini-Hochberg correction for testing multiple time points. C. PCoA of Bray-Curtis distances from stool communities collected from the three treatment groups over time (`r one_day_top_contrib_name`: R^2^ = `r top_R2`; `r one_day_sec_contrib_name`: R^2^ = `r sec_R2`). D. Shannon diversity in stool communities over time with colored lines representing the median value for each treatment group. For B-D, each symbol represents a sample from an individual mouse and symbol transparency corresponds to the day post-challenge that the sample was collected. E. Median relative abundances per treatment group for 6 out of the `r recovered_genera` genera that were affected by treatment, but recovered close to baseline levels by 7 days post-challenge (Fig. 3E, Data Set S1, Sheet X). Stool samples from either baseline and day 1 or baseline and day 7 were analyzed by paired Wilcoxan signed-rank test with Benjamini-Hochberg correction for testing all identified genera. The gray horizontal line represents the limit of detection.

``` {r figure_5_permanova}
fig_5_permanova_results = read_tsv(here("data/process/post_CDI_PEG_permanova_stools.tsv"))
r_sq_ordered_perm = fig_5_permanova_results %>% arrange(desc(R2)) %>%
  select(`effects`, `R2`, `Pr(>F)`) %>%
  filter(`Pr(>F)` < .05) #filter for sig
fig_5_permanova_sig_P_val <- r_sq_ordered_perm[,3] %>% sum()
fig_5_permanova_sig_R2 <- r_sq_ordered_perm[,2] %>% sum() #0.9280951

#Number of experiments represented in [post-CDI PEG subset
#Use same code from post-CDI_PEG_subset() in code/utilities.R to determine number of samples we had C. difficile CFU counts for per day
post_CDI_mice <- metadata %>%     
  filter(group == "C" & exp_num %in% c("M7","M9")| #Only use C mice from these experiments. Allocated groups to figures based on paper outline.
             group == "CWM" & exp_num %in% c("M6","M7", "M9")|
             group == "RM" & exp_num %in% c("M7","M9")|
             group == "FRM" & exp_num %in% c("M9"))
#Total number of experiments in post-CDI subset
post_CDI_total_exp <- post_CDI_mice %>%
  filter(!duplicated(exp_num)) %>% #Remove duplicate experiment numbers
  tally() #Total number of unique experiments
```

\includegraphics{figure_5.pdf}
**Figure 5. 1-day PEG treatment post *C. difficile* challenge prolongs colonization regardless of whether an FMT is also administered.** A. Setup off the experimental time line for experiments with post-CDI PEG treated mice. There were a total of 4 different treatment groups. All mice were administered 10 mg/kg clindamycin intraperitoneally (IP) 1 day before challenge with 10^3-5^ *C. difficile* 630 spores. 1. Did not receive any additional treatment. 2. Immediately after *C. difficile* challenge, mice received 15% PEG 3350 in the drinking water for 1 day. 3-4. 3-days after challenge, mice received 1-day PEG treatment and then received either a fecal microbiota transplant (3) or PBS (4) solution by oral gavage.  Mice were followed through 15-30 days post-challenge (only the post-CDI 1-day PEG group was followed through 30 days post-challenge). B. CFU/g of *C. difficile* stool measured over time via serial dilutions. The black line represents the limit of detection for the first serial dilution. C-D. Shannon diversity (C) and richness (D) in stool communities over time. B-D. Each symbol represents a stool sample from an individual mouse with the lines representing the median value for each treatment group. The transparancy of the symbol corresponds to the day post-challenge. Asterisks indicate time points with significant differences between groups by a Kruskall-Wallis test with a Benjamini-Hochberg correction for testing multiple times points. Colored rectangles indicates the 1-day PEG treatment period for applicable groups. The data presented are from a total of `r post_CDI_total_exp` separate experiments.
\newpage

\includegraphics{figure_6.pdf}
**Figure 6. For 1-day PEG treatment post *C. difficile* challenge mice that also receive an FMT only some bacterial genera were restored.** A. PCoA of Bray-Curtis distances from stool samples collected over time as well as the FMT solution that was administered to one treatment group. Each circle represents an individual sample, the transparency of the circle corresponds to day post-challenge as shown in Fig. 6C-D. B. Median relative abundances of 2 genera that were significantly different over multiple time points in mice that were administered either FMT or PBS solution via gavage C. Median relative abundances of the top 6 out of `r post_group_multi_day` genera that were significant over multiple timepoints, plotted over time (Data Set S1, Sheet X). For B-C, colored rectangles indicates the 1-day PEG treatment period for applicable groups. Gray horizontal lines represent the limit of detection.

\newpage

\includegraphics{figure_7.pdf}
**Figure 7. Specific microbiota features associated with prolonged *C. difficile* colonization in PEG treated mice.** A. Top ten bacteria that contributed to the random forest model trained on five day post-challenge community relative abundance data to predict wehther mice would still be colonized with *C. difficile* 10 days post-challenge. The median (point) and interquantile range (lines) change in AUROC when the bacteria is left out of the model is shown. B. The median relative abundances at 5 days post-challenge of the top ten bacteria that contributed to the random forest classification model. Color indicates whether the mice were still colonized with *C. difficile* 10 days post-challenge and the black horizontal line represents the median relative abundance. Each symbol represents a stool sample from an individual mouse and the shape of the symbol indicates whether the PEG-treated mice recieved a 5-day (Fig. 1-3), 1-day (Fig. 4) or post-CDI PEG (Fig. 5-6) treatment. C. The median relative abundances of the 5 genera with greater than 1% median relative abundance in the stool community over time. For B-C, the gray horizontal line represents the limit of detection.
\newpage

\includegraphics{figure_8.pdf}
**Figure 8. Schematic summarizing findings.** A. The gut microbiota of our C57Bl/6 mice is restant to *C. difficle* but treatment with either the antibiotic, clindamycin, or the osmotic laxative, PEG 3350 renders the mice susceptible to *C. difficile* colonization. Recovery of colonization resistance in clindamycin-treated mice is relatively straightforward and the mice clear *C.difficile* within 10 days post-challenge. However, for mice that recieved a 5-day PEG treatment or a 1-day PEG treatment after *C. difficile* challenge recovery of colonization resistance is more uncertain because mice were still colonized with *C. difficile* 30 days post-challenge in the case of several PEG treatments. We found increased *Porphyromonadaceae* and *Lachnospiraceae* were associated with recovery of colonization resistance, while increased *Enterobacteriaceae* and *Bacteroides* were associated with prolonged *C. difficle* colonization. B.Considering that most hospitals recommend not performing *C. difficile* testing on patients taking laxatives and laxatives are used when administering fecal microbiota transplants via colonoscopy to patients with recurrent CDIs. further studies are needed to evaluate if laxatives impact human microbiota colonization resistance. Further studies are needed to understand the impact of osmotic laxatives on *C. difficile* colonization resistance and clearance in human patients.
\newpage

\includegraphics{figure_S1.pdf}
**Figure S1. Microbiota dynamics post-challenge in the 5-day PEG treatment plus 10-day recovery mice.** A. *C. difficile* CFU/g over time in the stool samples collected from 5-day PEG treatment plus 10-day recovery mice. Same data presented in Fig. 1C, but the data for the other 3 treatment groups have been removed. B. Median relative abundances of 8 bacterial genera from day 0 post-challenge onward from the 10-day recovery mice. We analyzed samples from day 0 and day 8 post-challenge, which represented the the time points where mice were challenged with *C. difficile* and when the median relative *C. difficile* CFU stabilized for the group using the paired Wilcoxan signed-rank test, but no genera were signifcant after Benjamini-Hochberg correction.
\newpage

\includegraphics{figure_S2.pdf}
**Figure S2. PEG treatment still has a large impact on the mucosal microbiota 6 days post-challenge** A. The   
\newpage

\includegraphics{figure_S3.pdf}
**Figure S3. *C. difficile* challenge does not enhance the disruptive effect of PEG on the microbiota.** A.
\newpage

\includegraphics{figure_S4.pdf}
**Figure S4. Specific OTUs associated with clearance by 10 days post-challenge that are mostly absent in mice with prolonged *C. difficile* colonization. Ex. *Muribaculum intestinale*.** A.
\newpage

## References
