---
title: ''
csl: mbio.csl #Get themes at https://github.com/citation-style-language/styles
fontsize: 11pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
  html_document:
    df_print: paged
geometry: margin=1.0in
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE, tidy=TRUE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)
library(here)
library(readxl)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=2, nsmall=2, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)

metadata <- read_excel(here("data/process/metadata.xlsx"), col_types = c("text", "numeric", "text", "text", "numeric", "text", "numeric", "text", "text", "text", "text", "numeric", "numeric")) #specify column types

```



\vspace{35mm}

# An osmotic laxative renders mice susceptible to prolonged *Clostridioides difficile* colonization and hinders clearance

\vspace{35mm}


Sarah Tomkovich^1^, Ana Taylor^1^, Jacob King^1^, Joanna Colovas^1^, Lucas Bishop^1^, Kathryn McBride^1^, Sonya Royzenblat^1^, Nicholas A. Lesniak^1^, Ingrid L. Bergin^2^, Patrick D. Schloss^1$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

1\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI, USA

2\. The Unit for Laboratory Animal Medicine, University of Michigan, Ann Arbor, MI, USA


\newpage
\linenumbers


## Abstract
(Modify depending on target journal, currently abstract submitted to World Microbe Forum) Antibiotics are a major risk factor for *Clostridioides difficile* infections (CDIs) because of their impact on the intestinal microbiome. However, non-antibiotic medications such as the ubiquitous osmotic laxative polyethylene glycol (PEG) 3350, also alter the microbiota, but whether PEG impacts CDI susceptibility and clearance is unclear. To examine how PEG impacts susceptibility, we treated C57Bl/6 mice with 5-day and 1-day doses of 15% PEG in the drinking water and then challenged the mice with *C. difficile* 630 spores. We used clindamycin-treated mice as a control because they consistently clear *C. difficile* within 10 days post-infection (dpi). To examine how PEG treatment impacts clearance, we administered PEG for 1 day to clindamycin-treated, *C. difficile*-challenged mice either immediately following challenge or 3 dpi. We collected longitudinal stool samples to examine *C. difficile* levels in the stool via anaerobic culture and profiled the microbiota by 16S rRNA sequencing. PEG treatment alone was sufficient to render mice susceptible to CDI and 5-day PEG-treated mice remain colonized for up to 30 dpi. Additionally, 5-day PEG treated mice remained susceptible to CDI 10-days post treatment. In contrast, 1-day PEG treated mice were transiently colonized, clearing *C. difficile* within 7 dpi. Although 5-day PEG-treated mice exhibited prolonged *C. difficile* colonization, we saw no difference in histological inflammation between PEG- and clindamycin-treated mice. Additionally, administering PEG to mice after *C. difficile* challenge prolonged colonization up to 30 dpi in mice that received PEG immediately after challenge and 15 dpi in mice that received PEG 3 dpi. When we examined microbiota composition across our different treatment groups, we found decreased richness in the PEG-treated mice that exhibited prolonged *C. difficile* colonization. Importantly, there were increased Bacteroides and Enterobacteriaceae and decreased Lachnospiraceae and Oscillibacter in most of the PEG-treated mice with prolonged *C. difficile* colonization. Our findings suggest the osmotic laxative PEG 3350 alters the mouse microbiota and disrupts colonization resistance to *C. difficile*, as well as clearance in mice with a CDI. Considering that most hospitals recommend not performing *C. difficile* testing on patients taking laxatives and laxatives are used when administering fecal microbiota transplants via colonoscopy to patients with recurrent CDIs, further studies are needed to evaluate if laxatives impact human microbiota colonization resistance.

\newpage

## Introduction

Antibiotics are a major risk factor for *Clostridioides difficile* infections (CDIs) because they disrupt microbiota colonization resistance [@Britton2014]. However, antibiotics are not the only types of medications that disrupt the microbiota [@Maier2018; @LeBastard2017; @VichVila2020]. Although, other medications (proton pump inhibitors, osmotic laxatives, antimotility agents, and opioids) have been implicated as risk or protective factors for CDIs through epidemiological studies, whether the association is due to their impact on the microbiome is still unclear [@Oh2018; @Mora2012; @Nehra2018; @Krishna2013; @Tomkovich2019].

Many of the non-antibiotic medications associated with CDIs are known to modulate gastrointestinal motility leading to either increased or decreased colonic transit time, which in turn also strongly impacts microbiota composition and function [@Vandeputte2015; @VujkovicCvijin2020]. Stool consistency often serves as an approximation of intestinal motility. Our group has shown that when *C. difficile* negative controls are separated into two groups based on stool consistency, there are shared microbiota features such as lower alpha diversity in samples from CDI patients and control patients with diarrhea compared to control samples that were *C. difficile* negative with non-diarrheal consistency [@Schubert2014]. These results led to a hypothesis that bacterial communities from patients experiencing diarrhea are susceptible to developing CDIs.

Osmotic laxatives can lead to diarrhea depending on the administered dose and temporarily disrupt the human intestinal microbiota [@Nagata2019]. The ubiquitous osmotic laxative, polyethylene glycol (PEG) 3350 is found in Miralax, Nulytely, and Golytely and is also commonly used as bowel preparation for colonoscopies. Interestingly, previous studies have shown that treating mice with PEG alone altered microbiota composition, reduced acetate and butyrate production, altered the mucus barrier, and rendered the mice susceptible to *C. difficile* infection [@Kashyap2013; @Ferreyra2014; @Tropini2018; @VanInsberghe2020]. The mucus barrier is thought to mediate protection from *C. difficile* infections by protecting intestinal epithelial cells from the toxins produced by *C. difficile* [@Olson2013; @Diebel2014]. However, whether laxative results in more severe CDIs in mice and how long mice remain colonized with *C. difficile* after challenge is unclear.

Beyond susceptibility, PEG is also relevant in the context of treating recurrent CDIs via fecal microbiota transplant (FMT) where a healthy microbiota is administered to the patient to restore colonization. For FMTs that are delivered via colonoscopy, patients typically undergo bowel preparation by taking an osmotic laxative prior to the procedure. Many of the FMT studies to date rationalize the use of laxatives [@vanNood2013; @Razik2017; @Postigo2012] based on a 1996 case study with 2 pediatric patients where the authors suggested in the discussion that the laxative may help flush *C. difficile* spores and toxins from the intestine [@Liacouras1996].

In the past, our group has used C57BL6 mice to characterize how antibiotics including clindamycin disrupt the microbiota and influence *C. difficile* susceptibility and clearance [@Schubert2015; @Jenior2017; @Tomkovich2020]. Although, two groups have now shown PEG treatment alone renders mice susceptible to *C. difficile* [@Ferreyra2014; @VanInsberghe2020], these studies have raised additional questions regarding the dynamics and severity of infection as well as the role of laxative treatment in *C. difficile* clearance that should be addressed to better inform how we think about laxatives in the context of CDIs. Here, we used our C57BL/6 clindamycin model as a control group to characterize how long PEG-treated mice remain susceptible, whether PEG treatment results in more severe CDI and sustained *C. difficile* colonization, and if PEG treatment post-CDI can promote *C. difficile* clearance.

## Results

**5-day laxative treatment leads to prolonged *C. difficile* colonization in mice.** We compared mice treated with the osmotic laxative PEG 3350 to our standard 10 mg/kg clindamycin treatment, which temporarily renders the mice susceptible to *C. difficile*, with mice typically clearing *C. difficile* within 10 days post-infection [@Tomkovich2019; @Tomkovich2020]. All PEG-treated mice were administered a 15% PEG solution in the drinking water for 5-days, one group was also treated with clindamycin, and one group was allowed to recover for 10 days prior to challenge (Fig. 1A). PEG treatment resulted in weight loss in all 3 groups of PEG-treated mice, with the greatest change in weight observed on the fifth day of PEG treatment and the mice regained most of the weight five days after treatment (Fig. 1B). After either PEG, clindamycin, or PEG and clindamycin treatment all mice were challenged with 10^3^ *C. difficile* 630 spores (Fig. 1A). All treatments rendered mice susceptible to *C. difficile* colonization. However, PEG-treated mice remained colonized with *C. difficile* at high levels through thirty days post-infection (Fig. 1C). In contrast, the clindamycin-treated mice cleared *C. difficile* within ten days post-infection. Surprisingly, PEG-treated mice were still susceptible to *C. difficile* infection after 10-days of recovery from treatment although *C. difficile* was not detectable in most of the group in the initial five days post-infection (Fig. 1C). From 9 days post-infection onward, the median *C. difficile* stabilized for the 10-day recovery group of PEG-treated mice and remained high through 30 days post-infection (Fig. 1C). Thus, osmotic laxative treatment alone was sufficient to render mice susceptible to prolonged *C. difficile* colonization and PEG-treated mice remained susceptible for up to ten days post-treatment.

``` {r 5-day PEG fecal microbiota}
f_d_permanova <- read_tsv(here("data/process/5_days_PEG_permanova_stools.tsv"))
f_d_sign_contrib <- f_d_permanova %>% arrange(desc(R2)) %>%
  filter(!effects == "Total") %>% #Remove total since it is not one of the variables
  filter(`Pr(>F)` < 0.05) #Only select significant contributors

#List of variables that significantly explained variation in stool samples
f_d_top_effects <- f_d_sign_contrib %>%
  pull(effects)
#Sum of R2 values for variables that significantly explained variation in stool samples
f_d_top_R2_sum <- f_d_sign_contrib %>%
  select(R2) %>% sum
#P-value for variables that significantly explained variation in stool samples
f_d_top_p <- f_d_sign_contrib %>%
  select(`Pr(>F)`) %>% max # Both p values are the same, will be rounded in text

#Number of experiments represented in 5-day subset
#Use same code from five_day_PEG_subset() in code/utilities.R to determine number of samples we had C. difficile CFU counts for per day
five_day_mice <- metadata %>% filter(group == "C" & exp_num %in% c("M3","M4", "M5", "M8")| #Only use C mice from these experiments. Allocated groups to figures based on paper outline.
             group == "WM" & exp_num %in% c("M3","M4", "M5", "M8")|
             group == "WMC" & exp_num %in% c("M3","M4")|
             group == "WMR" & exp_num %in% c("M5","M6"))
#Total number of experiments in 5-day subset
five_day_total_exp <- five_day_mice %>%
  filter(!duplicated(exp_num)) %>% #Remove duplicate experiment numbers
  tally() #Total number of unique experiments

#PEG impacted genera
PEG_genera <- read_tsv(here("data/process/5_days_PEG_genus_PEG_paired.tsv")) %>%
  filter(p.value.adj < 0.05) %>% #Limit to significant genera
  tally()

#Genera that were different between treatment groups
f_d_group_genera <- read_tsv(here("data/process/5_days_PEG_genus_group_stools.tsv")) %>%
  filter(p.value.adj < 0.05)  #Select only significant p-values
f_d_group_total_genera <- f_d_group_genera %>%
  filter(!duplicated(genus))  %>% #Remove genera that appear multiple times
  tally()
#Genera that were different between treatment groups over multiple days
f_d_group_multi_day <- f_d_group_genera %>%
  filter(p.value.adj < 0.05) %>% #Select only significant p-values
  group_by(genus) %>%
  tally() %>%
  filter(n > 1) %>% #select genera that vary over multiple timepoints = n > 1
  arrange(desc(n)) %>%  #Rank by how many time each genera shows up
  nrow() #24 genera includes Peptostreptococcaceae and Unclassified

```
**5-day laxative treatment differentially disrupts the fecal microbiota compared to clindamycin treatment.** Since osmotic laxatives and clindamycin have previously been shown to disrupt the murine microbiota [@Kashyap2013; @Ferreyra2014; @Tropini2018; @VanInsberghe2020], we hypothesized the different *C. difficile* colonization dynamics between mice treated with the osmotic laxative or clindamycin were due to the two drugs having differential effects on the microbiota. We profiled the stool microbiota over time by sequencing the V4 region of the 16S rRNA gene to compare changes across treatment groups. We found time and treatment group explained half of the observed variation between fecal communities with most of the remaining variation explained by interactions between treatment group and other experimental variables including time, cage effects, and sequencing preparation (PERMANOVA combined R^2^ = `r f_d_top_R2_sum`, *P* `r format.pval(f_d_top_p, eps = 0.001)`, Fig. 2A, Data Set S1, Sheet X). Cage effects refer to the well-documented phenomenon that mice housed in the same cages have similar microbial communities due to copraphagy [@Nguyen2015], we tried to minimize the impact of cage effects on our experiment by breaking up cagemates when assigning mice to treatment groups and primarily housing only two mice per cage. Importantly, although we conducted a total of `r five_day_total_exp` separate experiments, the experiment number and it's interaction with treatment group was not one of the variables that significantly explained the observed variation in fecal communities (Data Set S1, Sheet X). Interestingly, none of the treatment groups recovered to their baseline community structure either 10 or 30 days post-infection suggesting other community features besides recovery to baseline were responsible for the prolonged *C. difficile* colonization in PEG-treated mice (Fig. 2B).

Next, we examined  alpha diversity by looking at Shannon diversity index over time. Although both clindamycin and PEG treatments decreased diversity, Shannon diversity was lower in the groups of mice that received PEG treatment compared to those that received clindamycin through thirty days post-infection (Fig. 2C). We next examined the bacterial genera that shifted after PEG treatment by comparing the baseline samples of mice treated with only PEG to samples from the same mice one day post-treatment. We found `r PEG_genera` genera that were altered by PEG treatment (Data Set S1, Sheet X). The majority of the bacterial relative abundances decreased after PEG, but *Enterobacteriaceae* and *Bacteroides* increased and the increase in *Bacteroides* was unique to PEG treatment as *Bacteroides* actually decreased in clindamycin treated mice (Fig. 2D). Finally, we examined the bacteria that differ across treatment groups over multiple timepoints. We found `r f_d_group_multi_day` genera were different over multiple timepoints out of the `r f_d_group_total_genera` that were different between treatment groups (Fig. 2E, Data Set S1, Sheet X). Thus, PEG has a significant impact on the fecal microbiota that was maintained over time and distinct from clindamycin treatment.

Surprisingly, *C. difficile* was not immediately detectable in the stools of the PEG-treated mice that were allowed to recover for 10 days prior to  challenge. We decided to examine the bacteria that changed during the post-infection period when the group median *C. difficile* shifted from undetectable at 1 day post-infection to detectable in the stool samples with the median stabilizing around 8 days post-infection (Fig. S1A). Interestingly, we found *Erysipelotrichaceae*, *Enterobacteriaceae*, and *Akkermansia* were changing during the time period when *C. difficile* was becoming detectable in the stools (Fig. S1B), although none of the bacteria were significant after multiple hypothesis correction (Data Set S1, Sheet X). Although we did not identify a clear signal to explain the delayed appearance of *C. difficile* in the 5-day PEG mice that were allowed to recover for 10 days prior to challenge, the delay is striking and could reflect changes in microbial activity or metabolites that were not examined in this study.

``` {r 5-day PEG tissue microbiota}
f_d_permanova_t <- read_tsv(here("data/process/5_days_PEG_permanova_tissues.tsv"))
f_d_sign_contrib_t <- f_d_permanova_t %>% arrange(desc(R2)) %>%
  filter(!effects == "Total") %>% #Remove total since it is not one of the variables
  filter(`Pr(>F)` < 0.05) #Only select significant contributors

#List of variables that significantly explained variation in tissue samples
f_d_top_effects_t <- f_d_sign_contrib_t %>%
  pull(effects)
#Sum of R2 values for variables that significantly explained variation in tissue samples
f_d_top_R2_sum_t <- f_d_sign_contrib_t %>%
  select(R2) %>% sum
#P-value for variables that significantly explained variation in tissue samples
f_d_top_p_t <- f_d_sign_contrib_t %>%
  select(`Pr(>F)`) %>% max # P value will be rounded in text

```
**5-day laxative treatment does not promote more severe CDIs despite altering the mucosal microbiota.** Given the findings from a previous study that demonstrated that PEG treatment disrupts the mucus layer and alters the immune response in mice [@Tropini2018], we decided to examine the impact of PEG treatment on the mucosal microbiota and CDI severity. To evaluate the mucosal microbiota, we sequenced snips of tissue collected from the cecum, proximal colon, and distal colon. Similar to what was observed with the stool samples, alpha diversity was lower in the PEG-treated mice compared to clindamycin treatment (Fig. 3A). Although alpha diversity continued to increase over time based on the communities from PEG-treated mice collected at 20 and 30 days post-infection (Fig. 3A, Data Set S1, Sheet X). Group, day of the experiment, and their interactions with other variables (cage, experiment number, and sample type) explained the majority of the variation observed in mucosal communities (PERMANOVA combined R^2^ = `r f_d_top_R2_sum_t`, *P* `r format.pval(f_d_top_p_t, eps = 0.05)`, Fig. 3B, Data Set S1, Sheet X). We saw the greatest difference in the mucosal microbiota between treatment groups at 6 days post-infection with 16 (insert code instead) genera that were significantly different in all three of the tissue types we collected (cecum, proximal colon, and distal colon; Fig. S3A, Data Set S1, Sheet X). However, at 30 days post-infection only *Bacteroides*, *Clostridiales*, *Firmicutes*, and *Ruminococcaceae* were different between treatment groups and just in the cecum tissues (Fig. 3C, Fig. 2E, Data Set S1, Sheet X). Interestingly, one of the genera that was different between treatment groups at 6 days post-infection was *Peptostreptococcaceae* (the genera with sequences that match C. difficile). *Peptostreptococcaceae* was primarily only present in the 5-day PEG treatment group of mice and decreased in the proximal and distal colon tissues over time (Fig. S2B). Thus, PEG treatment had a significant impact on the mucosal microbiota and we detected *C. difficile* sequences in the cecum, proximal colon, and distal colon tissue communities.

Next, we examined CDI severity by evaluating cecum and colon histopathology [@Reeves2011] and found there was no difference in cecum and colon scores between clindamycin and PEG-treated mice that were challenged with *C. difficile* at 4 days post-infection (Fig. 3D), the timepoint typically examined in *C. difficile* 630 challenged mice [@Dieterle2020]. We also looked at 6 days post-infection because that was when we started to see a large difference in C*. difficile* colonization levels between PEG- and clindamycin-treated mice (Fig. 1C). Although, there was a slight difference in the colon between PEG and clindamycin-treated mice, there was no difference in the cecum and the overall score is still relatively low given that the max possible summary score is 12 (Fig. 3E). Thus, although PEG treatment had a profound impact on the mucosal microbiota, the impact of *C. difficile* 630 infection on the cecum and colon was similar between PEG and clindamycin treated mice.

***C. difficile* challenge does not have a synergistic disruptive effect on the microbiota of PEG-treated mice** Because *C. difficile* itself can have an impact on the microbiota [@Jenior2018], we also sequenced the tissue and stools of mock-challenged clindamycin and 5-day PEG treated mice. Examining the stools of the mock-challenged mice revealed similar bacterial disruptions as the *C. difficile* challenged mice (Fig. S2A-C). Similarly, there was no difference between the tissues of mock and *C. difficile* challenged mice (Fig. S2D-F). Thus, most of the microbiota alterations we observed in the PEG-treated mice were a result of the laxative and not an interaction between the laxative and *C. difficile*.
``` {r 1-day PEG stool microbiota}

#Genera that were different between treatment days
#Genera that were changed by treatment
pt_d1 <- read_tsv(here("data/process/1_Day_PEG_genus_stats_PTtoD1.tsv")) %>%
  filter(p.value.adj < 0.05)
#Genera that were still different at 7 days post-infection  
pt_d7 <- read_tsv(here("data/process/1_Day_PEG_genus_stats_PTtoD7.tsv")) %>%
  filter(p.value.adj < 0.05)
pt_d7_genera <- pt_d7 %>% pull(genus)

recovered_genera <- pt_d1 %>%
  filter(!genus %in% pt_d7_genera) %>% #Remove genera that were still different at day 7
  nrow() #Tally number of rows


```
**1-day laxative treatment results in transient *C. difficile* colonization and minor microbiota disruption** Next, we decided to examine how a shorter osmotic laxative perturbation would impact the microbiome and susceptibility to *C. difficile*. We administered mice either a 1-day PEG treatment, a 1-day PEG treatment with a 1-day recovery period, or a clindamycin treatment before challenging them with *C. difficile 630* (Fig. 3A). In contrast to the 5-day PEG treated mice, the 1-day PEG treated mice were only transiently colonized and cleared *C. difficile* by 7 days post-infection (Fig. 3B). The stool communities of PEG-treated mice were also only transiently disrupted, with Shannon diversity close to baseline by 7 days post-infection (Fig. 3C-D). We found `r recovered_genera` bacteria were impacted by treatment but recovered close to baseline levels within 7 days post-infection including *Enterobacteriaceae*, *Clostridiales*, *Porpyromonadaceae*, and *Ruminococcaceae* (Fig. 3E, Data Set S1, Sheet X). These findings suggest the duration of treatment matters when considering the impact of the laxatives, with shorter treatments resulting in a transient loss of *C. difficile* colonization resistance.

**Post-CDI laxative treatment disrupts clearance in clindamycin-treated mice regardless of whether an FMT is also administered** Since 1-day PEG treatment resulted in a more mild microbiota perturbation, we decided to use the 1-day treatment to examine the hypothesis that PEG helps to flush *C. difficile* spores from the intestine. We were also interested in exploring whether PEG might help with engraftment in the context of FMTs. To examine the impact of PEG treatment on *C. difficile* clearance and FMT treatment, we treated 4 groups of mice with clindamycin and then challenged all mice with *C. difficile* before administering the following treatments: no additional treatment, 1-day PEG immediately after challenge, and 1-day PEG treatment 3 days after challenge followed by either administration of an FMT or PBS solution via gavage (Fig. 5A). In contrast to our hypothesis, all groups of mice that received PEG actually exhibited prolonged *C. difficile* colonization (Fig. 5B).

* Draft of results

  + Prolonged *C. difficile* colonization in post-CDI PEG treated mice (Fig. 5B)
  + FMT appears to partially restore alpha diversity (Shannon, but not richness Fig. 5C-D)
  + PCoA (necessary?) Could comment on the clustering of Clindamycin and PEG-treated mice that received FMT (Fig. 6A)
  + Only 2 genera significantly impacted by FMT treatment, likely not as important for clearance (Fig. 6B)
  + Bacteria that consistently differ between groups over time, associated with prolonged colonization (Fig. 6C)

``` {r ml results}
#AUC results
auc_results <-  read_csv(here("results/performance_results.csv")) %>%
  mutate(taxa_level = "genus") %>%  
  group_by(method) %>%
  summarize(median = median(AUC)) %>%
  arrange(desc(median))
#Highest AUC value
max_auc <- auc_results %>% select(median) %>% max()



```
**Five-day post-infection community data can predict mice that will have prolonged *C. difficile* colonization** After identifying bacteria associated with the 5-day, 1-day and post-CDI 1-day PEG treatments, we decided to examine the taxa that were influencing prolonged *C. difficile* colonization. We trained 3 types of machine learning models (random forest, logistic regression, and support vector machine) with input bacterial community data from 5 days post-infection to predict whether the mice were colonized with *C. difficile* 10 days post-infection. We chose 5 days post-infection because that was the earliest timepoint where we would see a treatment effect in the mice that were given 1-day PEG treatment three days after *C. difficile* challenge and then administered an FMT or PBS gavage. The random forest model had the highest performance (AUROC = `r max_auc`, Data Set S1, Sheet X), so we next performed permutation importance to examine the bacteria that were driving performance. We selected the top 10 bacteria contributing to our models performance (Fig. 7A) and examined their relative abundance at 5 days post-infection, the timepoint used to predict C. difficile colonization status on day 10 (Fig. 7B). Next, we focused on the 5 genera that were > 1 % relative abundance in either the cleared or colonized mice and examined the relative abundance dynamics of these bacteria over time. We found _, _, and _ tended to have a higher relative abundance and __, __ had a lower relative abundance in the mice with prolonged colonization compared to the mice that cleared *C. difficile* (Fig. 7C).
Previous work examining the impact of PEG on the murine microbiota found that PEG treatment resulted in the permanent loss of S24-7, also known as *Muribaculum intestinale* [@Tropini2018]. We decided to check our *Porphyromonadaceae* OTUs because *Muribaculum intestinale* is known to be classified as *Porphyromonadaceae* by the Ribosomal Database Project (RDP) database [@Vornhagen2020], *Porphyromonadaceae* was a top feature in the random forest model predicting prolonged *C. difficile* colonization, and had a high relative abundance in the communities of mice that cleared *C. difficile* within 10 days. We identified 4 OTUs that had at least (insert minimum percent identity) to *Muribaculum intestinale* and examined their abundance in mice that either cleared or were still colonized with *C. difficile* at 10 days post-infection (Fig. S4).

## Discussion

* Summary of major findings (Fig. 8A)

* Discussion of prolonged persistence. *C. difficile* sequences detected in tissue samples. Association with mucin-degrading bacteria suggested by recent papers.

* Discuss why we might not have observed more severe histology in PEG mice relative to clindamycin-treated mice
  + Antibiotics may also impact mucus layer
  + Strain of bacteria used

* Protective bacteria missing in PEG-treated mice

* Discuss what these findings might mean for human patients (Fig. 8B)
  + What's known regarding laxatives and susceptibility to CDIs
  + Clinical trial of PEG, results never posted [@Dieterle2018]
  + Relevance to human FMTs? Unclear what the best administration route is because there have been no studies designed to evaluate the best administration route for FMTs.


## Conclusions

## Acknowledgements
We thank members of the Schloss lab for feedback on planning the experiments and data presentation. We thank Andrew Henry for help with media preparation and bacterial culture and the Microbiology and Immunology department's postdoc association writing group members for their feedback on manuscript drafts. We also thank the Unit for Laboratory Animal Medicine at the University of Michigan for maintaining our mouse colony and providing the institutional support for our mouse experiments. Finally, we thank Kwi Kim, Austin Campbell, and Kimberly Vendrov for their help in maintaining the Schloss lab's anaerobic chamber. This work was supported by the National Institutes of Health (U01AI124255). ST was supported by the Michigan Institute for Clincial and Health Research Postdoctoral Translation Scholars Program (UL1TR002240 from the National Center for Advancing Translational Sciences).

## Materials and Methods
* Histopathology [@Theriot2011]

\newpage
```{r sample_size for figure legends}

#Use same code from five_day_PEG_subset() in code/utilities.R to determine number of samples we had C. difficile CFU counts for per day
five_day_mice <- metadata %>% filter(group == "C" & exp_num %in% c("M3","M4", "M5", "M8")| #Only use C mice from these experiments. Allocated groups to figures based on paper outline.
             group == "WM" & exp_num %in% c("M3","M4", "M5", "M8")|
             group == "WMC" & exp_num %in% c("M3","M4")|
             group == "WMR" & exp_num %in% c("M5","M6"))
five_d_cfu <- five_day_mice %>%
  filter(sample_type == "stool") %>% #Select only stool samples
  filter(day %in% c(0, 1, 2, 3, 4, 5, 6, 8, 9, 10, 15, 20, 25, 30)) %>%  #select timepoints on plot
  filter(!is.na(avg_cfu)) %>% #Remove rows with NA values
  group_by(day) %>%
  count()
#Figure out min & max number of samples we had cfu counts for
five_d_cfu_n_min <- min(five_d_cfu$n)
five_d_cfu_n_max <- max(five_d_cfu$n)

```

\includegraphics{figure_1.pdf}

**Figure 1. 5-day PEG treatment prolongs susceptibility and mice become persistently colonized with *C. difficile*.** A. Setup of the experimental timeline for experiments with 5-day PEG treated mice. Clindamycin was administered at 10 mg/kg by intraperitoneal injection. 15% PEG 3350 was administered in the drinking water for five days. B. Weight change from baseline weight in groups after treatment with PEG and/or clindamycin, followed by *C. difficile* challenge. C. *C. difficile* CFU/gram stool measured over time (N = `r five_d_cfu_n_min`-`r five_d_cfu_n_max` mice per timepoint) via serial dilutions. The black line represents the limit of detection for the first serial dilution. CFU quantification data was not available for each mouse due to stool sampling difficulties (particularly the day the mice came off of the PEG treatment) or early deaths. For B-C, lines represent the median for each treatment group and circles represent samples from individual mice. Asterisks indicate timepoints where the weight change or CFU/g was significantly different between groups by the Kruskal-Wallis test with Benjamini-Hochberg correction for testing multiple timepoints. The data presented are from a total of `r five_day_total_exp` separate experiments.
\newpage

\includegraphics{figure_2.pdf}
**Figure 2. 5-day PEG treatment disrupts the stool microbiota for a longer amount of time compared to clindamycin-treated mice.** A. PCoA of Bray-Curtis distances from stool samples collected throughout the experiment.
\newpage

\includegraphics{figure_3.pdf}
**Figure 3. 5-day PEG treatment does not result in more severe CDIs, although mucosal microbiota is altered.** A.
\newpage

``` {r figure_4_permanova}
one_day_permnova_results <- read_tsv(here("data/process/1_day_PEG_permanova.tsv"))
one_day_top_2_contrib <- one_day_permnova_results %>% arrange(desc(R2)) %>%
  top_n(3, R2)
one_day_top_contrib_name = (one_day_top_2_contrib[2,1])
one_day_sec_contrib_name = (one_day_top_2_contrib[3,1])
top_R2 = one_day_top_2_contrib[2,6]
sec_R2 = one_day_top_2_contrib[3,6]


#Use same code from one_day_PEG_subset() in code/utilities.R to determine number of samples per day
one_day_samples_per_day <- metadata %>%
  filter(group == "C" & exp_num %in% c("M6")| #Only use C mice from this experiments. Allocated groups to figures based on paper outline.
             group == "1RM1" & exp_num %in% c("M6R")| #Had to differentiate experiment 6 from 6R in the metadata to create unique_mouse_id that wouldn't overlap for the M1 & 1RM1 mice that are both labeled with mouse_ids that are #s1-6
             group == "M1" & exp_num %in% c("M6")) %>%
  group_by(day) %>%
  count()
#To Do: this code is not specific for CFU values or the timepoints shown on panel B
one_day_sample_min <- min(one_day_samples_per_day$n)
one_day_sample_max <- max(one_day_samples_per_day$n)


```
\includegraphics{figure_4.pdf}
**Figure 4. 1-day PEG treatment renders mice susceptible to transient *C. difficile* colonization.** A. Setup of the experimental timeline for the 1-day PEG treated subset of mice. B. CFU/gram stool measured over time (N = `r one_day_sample_min`-`r one_day_sample_max` mice per timepoint) via several dilutions. The black dotted line represents the limit of detection for the first serial dilution. Asterisks indicate timepoints where the CFU/gram was significantly different between groups using the Kruskall-Wallis test with a Benjamini-Hochberg correction for multiple timepoints. C. Principle Coordinate Analysis plot of the groups over time with the alpha representating the same time scale as in panel D (`r one_day_top_contrib_name`: R^2^ = `r top_R2`; `r one_day_sec_contrib_name`: R^2^ = `r sec_R2`). D. Shannon diversity Index of the groups over time.  Only days with samples from all groups are shown. E. Line plots of relative percent abundance of selected genera over time. Only days with samples from all groups shown. The gray line represents the limit of detection.

``` {r figure_5_permanova}
fig_5_permanova_results = read_tsv(here("data/process/post_CDI_PEG_permanova_stools.tsv"))
r_sq_ordered_perm = fig_5_permanova_results %>% arrange(desc(R2)) %>%
  select(`effects`, `R2`, `Pr(>F)`) %>%
  filter(`Pr(>F)` < .05) #filter for sig
fig_5_permanova_sig_P_val <- r_sq_ordered_perm[,3] %>% sum()
fig_5_permanova_sig_R2 <- r_sq_ordered_perm[,2] %>% sum() #0.9280951
```

\includegraphics{figure_5.pdf}
**Figure 5. 1-day PEG treatment post *C. difficile* challenge prolongs colonization regardless of whether an FMT is also administered.** All mice were administered 10 mg/kg clindamycin intraperitoneally (IP) 1 day before challenge with *C. difficile* 630 spores on day 0. 15% PEG 3350 was administered in the drinking water for 1 day. Mice feces were collected daily through the end of the experiment (30 days post-infection for Clind. + 1-day PEG group, 15 days post-infection for all others). CFU quantification data was not available for each mouse due to stool sampling difficulties (particularly the day the mice came off of the PEG treatment) or early deaths. B. CFU/g of *C. difficile* stool measured over time via serial dilutions. C. Median and individual Shannon Diversity measured over time. D. Median and individual richness measured over time. B-D. Asterisks indicate time points with significant differences between groups using Kruskall-Wallis tests with a Benjamini-Hochberg correction for testing multiple times points. Black line represents the limit of detection for the first serial dilution. Background shading indicates PEG treatment period for applicable groups. Opacity of points reflects time point. The data presented are from a total of *** separate experiments.
\newpage

\includegraphics{figure_6.pdf}
**Figure 6. For 1-day PEG treatment post *C. difficile* challenge mice that also receive an FMT only some bacterial genera were restored.** A. PCoA of all groups + FMT plotted over time. Opacity of points reflects time point. B. Relative abundance of genera significantly different over multiple time points post FMT or PBS treatment, plotted over time. Limit of detection is .1%. C. Relative abundances of top 6 significant genera ranked by number of days significant, plotted over time. Limit of detection is .1%.

\newpage

\includegraphics{figure_7.pdf}
**Figure 7. Specific microbiota features associated with prolonged *C. difficile* colonization in PEG treated mice.** A.
\newpage

\includegraphics{figure_8.pdf}
**Figure 8. Schematic summarizing findings.** A.
\newpage

\includegraphics{figure_S1.pdf}
**Figure S1. 5-day PEG treatment plus 10-day recovery mice microbiota dynamics post-infection.** A.
\newpage

\includegraphics{figure_S2.pdf}
**Figure S2. Impact of PEG treatment on the mucosal microbiota is greatest 6 days post-infection**  
\newpage

\includegraphics{figure_S3.pdf}
**Figure S3. *C. difficile* challenge does not enhance the disruptive effect of PEG on the microbiota.** A.
\newpage

\includegraphics{figure_S4.pdf}
**Figure S4. Specific OTUs associated with clearance that are mostly absent in mice with prolonged *C. difficile* colonization. Ex. *Muribaculum intestinale*.** A.
\newpage

## References
